README.md

```diff
@@ -222,7 +223,19 @@ Remove the asset(s) (`assets`) from holding and send a `WithdrawAsset` XCM messa
   be executed on them. There will typically be only one valid location on any given asset/chain combination.
 - `effects: Vec<Order>`: The orders to execute on the assets once withdrawn *on the reserve location*.
 
+<<<<<<< HEAD
 Errors:
+=======
+### `AccountClaim`
+Sends a claim verified to be from the `source` to the `destination` on the parachain. It has an `expiration` timestamp and a `scope` used to define the scope under which such a `claim` is made. A chain should only send such a claim if it is verified to have been intended to be sent by the `source`. It facilitates authentication across parachains.
+
+- `source: MultiLocation`
+- `destination: MultiLocation`
+- `timestamp: Compact<u48>`
```

Comment:
How is timestamp going to be used? Upon receive the message, the dest chain should just transfer the assets into the destination account immediately.

Reply:
This is not used for assets but merely indicates the validity of the claim. In the crowdloan example it would you would first sign an extrinsic on the relay chain instructing it to send a claim to the parachain with such a claim and an expiration date. Until such a timestamp has passed, the parachain would allow the user to then submit an extrinsic to withdraw their reward from the `destination` account.